### ISSUE TITLE
Evaluate inline code in cached documentation mode

### ISSUE THREAD

**Issue by brandonwillard** [mpastell/Pweave#17]
Is there any way to include inline code when using the caching feature?

**Comment by mpastell** [comment]
No, that is not currently implemented. Do you think it would be an important feature?

**Comment by brandonwillard** [comment]
Yes.  Actually, I'm a little surprised that they're excluded.  It's likely that I'm not understanding the purpose of documentation mode, but it seems to do the caching when I use it and not otherwise.

Anyway, I need the caching features (ideally to the extend implemented by [knitr](http://yihui.name/knitr/)) in general, and, since I'm printing LaTeX expressions generated by [SymPy](sympy.org), inline evaluation can often be a reasonable approach.
